src/main/kotlin/com/cognifide/gradle/aem/common/instance/Instance.kt

```diff
@@ -53,6 +53,9 @@ open class Instance(@Transient @get:JsonIgnore protected val aem: AemExtension)
         else -> user to password
     }
 
+    @get:JsonIgnore
```

Comment:
hmmm use provider API; i recently refactored the code here

Reply:
On it